raiden/routing.py

```diff
@@ -256,7 +255,7 @@ def get_best_routes_pfs(
         log_message = e.args[0]
         log_info = e.args[1] if len(e.args) > 1 else {}
         log.warning("An error with the path request occured", log_message=log_message, **log_info)
-        return False, [], None
+        return log_message, [], None
```

Comment:
main change is here where `log_message` is the `ServiceRequestFailed` exception message that is raised by `query_paths`.